### PR TITLE
cleanup(palette v3): button v2 👋 

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **Enhancement**

This PR resolves []

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

### Changelog updates

#nochangelog
#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

File: src/lib/Components/Artist/ArtistAbout/ArtistAboutShows.tsx [hunk 1]
```diff
@@ -58,12 +58,7 @@ const ArtistAboutShows: React.FC<Props> = ({ artist }) => {
           contentContainerStyle={{ paddingBottom: 15, paddingLeft: 20 }}
         />
         {!!pastShows.length && (
-          <Button
-            variant={"secondaryGray"}
-            onPress={() => navigate(`/artist/${artist?.slug!}/shows`)}
-            size="medium"
```
Comment (pvinis) [inline src/lib/Components/Artist/ArtistAbout/ArtistAboutShows.tsx:64]: using the mappings to change the size and variants.

File: src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx [hunk 2]
```diff
@@ -220,7 +220,7 @@ export const SavedSearchBanner: React.FC<SavedSearchBannerProps> = ({
 
   return (
     <Button
-      variant={enabled ? "secondaryOutline" : "primaryBlack"}
```
Comment (pvinis) [inline src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx:223]: same. renaming everything to the mapping we had in buttonv2 file.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ✅ **No changelog changes**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 3364e864361ef8da9993ae9fd06b7e1d03894bab
</p>

File: src/palette/elements/Button/index.tsx [hunk 3]
```diff
@@ -1,87 +1 @@
-import { usePaletteFlagStore } from "palette/PaletteFlag"
```
Comment (pvinis) [inline src/palette/elements/Button/index.tsx:1]: this whole file is simplified, to just export the button stuff.

removed all the v2 buttons and mappings.

**Review by olerichter00 — APPROVED** [review]
Very nice 🥇

**Review by AlicanAkyuz — APPROVED** [review]
looking handsome

**Comment by MounirDhahri** [comment]
🎉

**Pull request merged**